news/2022-09-11-new-featured-artist-aaaa.md

```diff
@@ -0,0 +1,155 @@
+---
+layout: post
+title: "New Featured Artist: AAAA"
+date: 2022-09-11 16:30:00 +0000
+---
+
+WELCOME **AAAA** TO OUR FEATURED ARTIST LIBRARY.
+
+![](https://assets.ppy.sh/artists/301/header.jpg)
+
+Earning the spotlight in games like *Arcaea*, *maimai*, *CHUNITHM*, *pop'n music*, *SDVX*, *Muse Dash*, and... more than we can reasonably list, **AAAA** is an undeniable legend in the realm of rhythm game compositions.
+
+**22** insanely awesome tracks are ready for mapping from [**AAAA**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/301)!
+
+**AAAA**'s Featured Artist reveal was a bit unorthodox. In collaboration with fellow legends [Hino Isuka](https://osu.ppy.sh/beatmaps/artists/272) and [Umeboshi Chazuke](https://osu.ppy.sh/beatmaps/artists/98), **AAAA** helped create the custom song [*steganography*](LINK) for the MWC 4K 2022 tiebreaker!
```

Comment:
It's missing a link here (to the song file/listing?)

Reply:
unsure if markdown supports greek letters but the e in steganography is actually epsilon "ε"

Reply:
iirc news posts use the romanised title